composer.json

```diff
@@ -12,11 +12,18 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=7.2",
-        "prismic/php-sdk": "dev-master"
+        "prismic/php-sdk": "dev-master",
+        "magento/framework": "*"
     },
     "replace": {
         "elgentos/prismic-io": "*"
     },
+    "repositories": [
```

Comment:
Why add the Magento repo here?

Reply:
Because it's a start to add some additional Unit testing!
Which we can run on future MR

`magento/framework` isn't in the packagist.org